### PR TITLE
Added Jio-Saavn

### PR DESCRIPTION
**Pull request by yashpalgoyal1304**
... in fastlane full_description.txt

Also, dont push tag just for getting fdroid to pick it ;)

related: 
https://github.com/Shabinder/SpotiFlyer/commit/ce268c3fe8965b2ed22e1006d7ed8c6f8b3b35c9
https://github.com/Shabinder/SpotiFlyer/search?q=jio&type=commits

### CODE REVIEW

**Comment by Shabinder** [comment]
Hey @yashpalgoyal1304 thanks for your contribution , what do you mean that don't push tags for Fdroid to pick up?

Fdroid does build new apk after a tag is made...right?

**Comment by yashpalgoyal1304** [comment]
sorry, i had a misunderstanding. yeah, fdroid does build it, but i was saying that it will feel odd to have a new version now just for this small thing.

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -10,7 +10,8 @@ Supported Platforms:
  - Gaana
  - Youtube
  - Youtube Music
+ - Jio-Saavn
  - (more coming soon)
 
 Note:
- - The availability of YouTube Music in your country is IMPORTANT, if it isn't available consider using a VPN.
\ No newline at end of file
+ - The availability of YouTube Music in your country is IMPORTANT, if it isn't available consider using a VPN.
```
Comment (yashpalgoyal1304) [inline fastlane/metadata/android/en-US/full_description.txt:17]: also, i havent changed anything in this line. still it's shown. have u faced similar thing before?

**Comment by Shabinder** [comment]
> sorry, i had a misunderstanding. yeah, fdroid does build it, but i was saying that it will feel odd to have a new version now just for this small thing.

Yeah I agree and hence I am not going to create a tag only for this change , instead this will be incorporated in next version. 👍

**Pull request merged**